### PR TITLE
Cumulus 970 - add logging for intermittent unit test fail

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## Test Plan
Things that should succeed before merging.

- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

## Release PR

If this is a **release** PR, make sure you branch name starts with `release-` prefix, e.g. `release-v-1.1.2`.

Reviewers: @tag-your-friends

### CODE REVIEW

File: tasks/move-granules/tests/index-test.js [hunk 1]
```diff
@@ -160,33 +160,71 @@ test.serial('should overwrite files', async (t) => {
...
     t.context.publicBucket,
     destKey
   );
 
+  console.log('CUMULUS-970 debugging: headobject complete');
+
   // re-stage source jpg file with different content
   const content = randomString();
+
+  const uploadParams2 = {
```
Comment (markdboyd) [inline tasks/move-granules/tests/index-test.js]: We could just do `uploadParams.Body = content` here and pass `uploadParams` to `promiseS3Upload`

File: tasks/move-granules/tests/index-test.js [hunk 2]
```diff
@@ -160,33 +160,71 @@ test.serial('should overwrite files', async (t) => {
...
 
+  const uploadParams = {
+    Bucket: t.context.stagingBucket,
+    Key: sourceKey,
+    Body: 'Something'
+  };
+
+  console.log(`CUMULUS-970 debugging: start s3 upload. params: ${JSON.stringify(uploadParams)}`);
+
   await promiseS3Upload({
```
Comment (markdboyd) [inline tasks/move-granules/tests/index-test.js]: Might as well just pass in `uploadParams` here

File: tasks/move-granules/tests/index-test.js (hunk 1 above)
Comment (laurenfrederick) [reply tasks/move-granules/tests/index-test.js]: done

File: tasks/move-granules/tests/index-test.js (hunk 2 above)
Comment (laurenfrederick) [reply tasks/move-granules/tests/index-test.js]: done

**Pull request merged**